js_modules/dagit/packages/core/src/nav/__tests__/useDaemonStatus.test.tsx

```diff
@@ -0,0 +1,247 @@
+import {MockedProvider} from '@apollo/client/testing';
+import {act, render, screen, waitFor} from '@testing-library/react';
+import * as React from 'react';
+
+import {InstigationStatus} from '../../graphql/types';
+import {useDaemonStatus} from '../../instance/useDaemonStatus';
+import {WorkspaceProvider} from '../../workspace/WorkspaceContext';
+import {
+  buildInstanceWarningQuery,
+  buildWorkspaceQueryWithNoSchedulesOrSensors,
+  buildWorkspaceQueryWithScheduleAndSensor,
+} from '../__fixtures__/useDaemonStatus.fixtures';
+
+describe('useDaemonStatus', () => {
+  const Test = () => {
+    const status = useDaemonStatus();
+    if (!status) {
+      return <div>No status</div>;
+    }
+    return (
+      <>
+        <div>Type: {status.type}</div>
+        <div>Content: {status.content}</div>
+      </>
+    );
+  };
```

Comment:
use renderHook instead of making a custom Test component

example usage: https://github.com/dagster-io/dagster/blob/3d97e2ec2226de3bc3107df1c4edfb23ca963d6b/js_modules/dagit/packages/core/src/runs/__tests__/RunFilterInput.test.tsx#L55

Reply:
Docs: https://react-hooks-testing-library.com/reference/api#renderhook